src/Commands/ServiceBus/Topic/SubscriptionDetailsCommand.cs

```diff
@@ -0,0 +1,129 @@
+
+    protected override string GetCommandName() => "details";
+
+    protected override string GetCommandDescription() =>
+        """
+        Get details about a Service Bus subscription. Returns subscription runtime properties including message counts, delivery settings, and other metadata.
+
+        Required arguments:
+        - namespace: Service Bus namespace name. (This is usually in the form <namespace>.servicebus.windows.net)
+        - topic-name: Topic name containing the subscription
+        - subscription-name: Name of the subscription to get details for
+        """;
+
+    protected override void RegisterOptions(Command command)
+    {
+        base.RegisterOptions(command);
+        command.AddOption(_namespaceOption);
+        command.AddOption(_topicOption);
+        command.AddOption(_subscriptionNameOption);
+    }
+
+    protected override void RegisterArguments()
+    {
+        base.RegisterArguments();
+        AddArgument(CreateSubscriptionNameArgument());
+        AddArgument(CreateTopicNameArgument());
+        AddArgument(CreateNamespaceArgument());
+    }
+
+    protected override SubscriptionDetailsArguments BindArguments(ParseResult parseResult)
+    {
+        var args = base.BindArguments(parseResult);
+        args.Namespace = parseResult.GetValueForOption(_namespaceOption);
+        args.TopicName = parseResult.GetValueForOption(_topicOption);
+        args.SubscriptionName = parseResult.GetValueForOption(_subscriptionNameOption);
+        return args;
+    }
+
+    [McpServerTool(Destructive = false, ReadOnly = true)]
+    public override async Task<CommandResponse> ExecuteAsync(CommandContext context, ParseResult parseResult)
+    {
+        var args = BindArguments(parseResult);
+
+        try
+        {
+            if (!await ProcessArguments(context, args))
+            {
+                return context.Response;
+            }
+
+            var service = context.GetService<IServiceBusService>();
+            var details = await service.GetSubscriptionDetails(
+                args.Namespace!,
+                args.TopicName!,
+                args.SubscriptionName!,
+                args.Tenant,
+                args.RetryPolicy);
+
+            context.Response.Results = ResponseResult.Create(
+                new SubscriptionDetailsCommandResult(details),
+                ServiceBusJsonContext.Default.SubscriptionDetailsCommandResult);
+        }
+        catch (Exception ex)
+        {
+            HandleException(context.Response, ex);
+        }
+
+        return context.Response;
+    }
+
+    protected override string GetErrorMessage(Exception ex) => ex switch
+    {
+        ServiceBusException exception when exception.Reason == ServiceBusFailureReason.MessagingEntityNotFound =>
+            $"Topic or subscription not found. Please check the topic and subscription names and try again.",
```

Comment:
Should we have an aka.ms link to a TSG section for any of these new tools or should we wait and see issue feedback first

Reply:
The base method that calls GetErrorMessage tacks the aka.ms link on.